pypeit/par/pypeitpar.py

```diff
@@ -3109,6 +3111,10 @@ def __init__(self, filt_iter=None, sobel_mode=None, edge_thresh=None, sobel_enha
         dtypes['use_maskdesign'] = bool
         descr['use_maskdesign'] = 'Use slit-mask designs to identify slits.'
 
+        defaults['maskdesign_filename'] = None
+        dtypes['maskdesign_filename'] = [str, list]
```

Comment:
Not sure if having `None` would violate the current `dtype` requirements and raise an error. Do we need to add `None` to the `dtype` list?

Reply:
this one is ok